CoreHelp/src/au/gov/asd/tac/constellation/help/HelpServlet.java

```diff
@@ -60,14 +60,14 @@
             ".txt", "text/plain",
             ".md", "text/html" // this allows a markdown file be converted to html on the fly
     );
-    private static boolean wasRedirect = false;
+    private static boolean redirect = false;
 
-    protected boolean getWasRedirect() {
-        return wasRedirect;
+    protected boolean isRedirect() {
+        return redirect;
     }
 
-    protected void setWasRedirect(final boolean wasRedirect) {
-        this.wasRedirect = wasRedirect;
+    protected static void setWasRedirect(final boolean wasRedirect) {
+        HelpServlet.redirect = wasRedirect;
```

Comment:
bit confused by this one since the getter isn't static

Reply:
Not sure why it only picked up code smells on the setter. These are the code smells it wanted changed. 
![image](https://user-images.githubusercontent.com/73083035/141860160-6ca755bd-04f3-4279-b009-764fed020190.png)

Reply:
Right so the variable is actually static hence the smells but yes it is rather odd that the getter wasn't picked up. Unsurprisingly, if you use the netbeans shortcuts to make the getter, it makes the method static so I wonder whether it simply hasn't realised for some reason